core-bundle/src/Command/MigrateCommand.php

```diff
@@ -0,0 +1,266 @@
+                    $first = false;
+                }
+
+                $this->io->writeln(' * '.$migration);
+            }
+
+            foreach ($this->getRunOnceFiles() as $file) {
+                if ($first) {
+                    $this->io->section('Pending migrations');
+                    $first = false;
+                }
+
+                $this->io->writeln(' * Runonce file: '.$file);
+            }
+
+            if ($first) {
+                break;
+            }
+
+            if (!$this->io->confirm('Execute the listed migrations?')) {
+                return false;
+            }
+
+            $this->io->section('Execute migrations');
+
+            foreach ($this->migrations->runMigrations() as $result) {
+                $this->io->writeln(' * '.$result->getMessage());
+            }
+
+            foreach ($this->getRunOnceFiles() as $file) {
+                $this->executeRunonceFile($file);
+                $this->io->writeln(' * Executed runonce file: '.$file);
+            }
+
+            $this->io->success('Executed migrations.');
+        }
+
+        return true;
+    }
+
+    private function getRunOnceFiles(): array
+    {
+        try {
+            $files = $this->fileLocator->locate('config/runonce.php', null, false);
+        } catch (FileLocatorFileNotFoundException $e) {
+            return [];
+        }
+
+        return array_map(function ($path) {
+            return rtrim((new Filesystem())->makePathRelative($path, $this->projectDir), '/');
+        }, $files);
+    }
+
+    private function executeRunonceFile(string $file): void
+    {
+        $this->framework->initialize();
+
+        include $this->projectDir.'/'.$file;
+
+        (new Filesystem())->remove($this->projectDir.'/'.$file);
+    }
+
+    private function executeSchemaDiff(bool $completeOption): bool
+    {
+        if (null === $this->installer) {
+            $this->io->error('Service contao.installer of contao/installation-bundle not found.');
+
+            return false;
+        }
+
+        $commandsByHash = [];
+
+        while (true) {
+            $this->installer->compileCommands();
+            $commands = $this->installer->getCommands();
+
+            if (!$commands) {
+                return true;
+            }
+
+            $hasNewCommands = \count(
+                array_filter(
+                    array_keys(
+                        array_merge(...array_values($commands))
+                    ),
+                    static function ($hash) use ($commandsByHash) {
+                        return !isset($commandsByHash[$hash]);
+                    }
+                )
+            );
+
+            if (!$hasNewCommands) {
+                return true;
+            }
+
+            $this->io->section('Pending database migrations');
+
+            $commandsByHash = array_merge(...array_values($commands));
+
+            $this->io->listing($commandsByHash);
+
+            $options = ['yes', 'yes, with delets', 'no'];
```

Comment:
```suggestion
            $options = ['yes', 'yes, with deletes', 'no'];
```

Reply:
I noticed that during they keynote too 😁